apps/console/src/features/identity-providers/components/wizards/organization-enterprise/organization-enterprise-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,255 @@
+    /**
+     * Trigger form submit.
+     * @param {() => void} submitFunctionCb - Callback.
+     */
+    triggerSubmission: (submitFunctionCb: () => void) => void;
+    /**
+     * Trigger previous page.
+     * @param {() => void} previousFunctionCb - Callback.
+     */
+    triggerPrevious: (previousFunctionCb: () => void) => void;
+    /**
+     * Callback to change the wizard page,
+     * @param {number} pageNo - Page Number.
+     */
+    changePageNumber: (pageNo: number) => void;
+    /**
+     * IDP template.
+     */
+    template: IdentityProviderTemplateInterface;
+    /**
+     * Total wizard page count.
+     * @param {number} pageCount - Page number.
+     */
+    setTotalPage: (pageCount: number) => void;
+    /**
+     * Callback to be triggered for form submit.
+     * @param values
+     */
+    onSubmit: (values: OrganizationEnterpriseAuthenticationProviderCreateWizardFormErrorValidationsInterface) => void;
+}
+
+/**
+ * Authentication Provider Create Wizard content component.
+ *
+ * @param {OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface} props - Props injected to the component.
+ *
+ * @return {React.ReactElement}
+ */
+export const OrganizationEnterpriseAuthenticationProviderCreateWizardContent:
+    FunctionComponent<OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface> = (
+        props: OrganizationEnterpriseAuthenticationProviderCreateWizardContentPropsInterface
+    ): ReactElement => {
+
+        const {
+            triggerSubmission,
+            triggerPrevious,
+            changePageNumber,
+            template,
+            setTotalPage,
+            onSubmit,
+            ["data-componentid"]: componentId
+        } = props;
+
+        const { t } = useTranslation();
+
+        const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+        const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+        /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+        useEffect(() => {
+
+            getIDPlist();
+        }, []);
+
+        /**
+     * Get Idp List.
+     */
+        const getIDPlist = (): void => {
+
+            setIdPListRequestLoading(true);
+
+            getIdentityProviderList(null, null, null)
+                .then((response) => {
+                    setIdPList(response);
+                })
+                .catch((error) => {
+                    handleGetIDPListCallError(error);
+                })
+                .finally(() => {
+                    setIdPListRequestLoading(false);
+                });
+        };
+
+        /**
+     * Check whether IDP name is already exist or not.
+     *
+     * @param value IDP name - IDP Name.
+     *
+     * @returns error msg if name is already taken.
+     */
+        const idpNameValidation = (value): string => {
+
+            let nameExist = false;
```

Comment:
Shall we type this?

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3176